qiskit/providers/aer/jobs/aerjob.py

```diff
@@ -24,13 +24,19 @@
 
 
 class AerJob(Job):
-    """AerJob class.
-
-    Attributes:
-        _executor (futures.Executor): executor to handle asynchronous jobs
-    """
+    """AerJob class for Qiskit Aer Simulators."""
 
     def __init__(self, backend, job_id, fn, qobj, executor=None):
+        """ Initializes the asynchronous job.
+
+        Args:
+            backend(AerBackend): the backend used to run the job.
+            job_id(str): a unique id in the context of the backend used to run the job.
+            fn(function): callabled function.
```

Comment:
```suggestion
            fn(function): callable function.
```

We should probably describe what this function is for, and what inputs/outputs it should expect.